client/src/profiles.rs

```diff
@@ -109,11 +113,25 @@ impl Profile {
         tokio::task::block_in_place(|| fs::unzip(&profile))?;
 
         #[cfg(unix)]
-        set_permissions(vec![
-            &profile.directory.join(consts::VOXYGEN_FILE),
-            &profile.directory.join(consts::SERVER_CLI_FILE),
-        ])
-        .await?;
+        {
+            let voxygen_file = profile.directory.join(consts::VOXYGEN_FILE);
+            let server_cli_file = profile.directory.join(consts::SERVER_CLI_FILE);
+
+            let os_release = Path::new("/etc/os-release");
```

Comment:
With the suggestion from above in mind maybe can be contained in `nix::is_nix_host()`.